packages/manager/apps/dedicated/client/app/account/user/components/newAccountForm/field/new-account-form-field-component.controller.js

```diff
@@ -9,21 +9,38 @@ import {
   PHONE_PREFIX,
   MODEL_DEBOUNCE_DELAY,
   FIELD_NAME_LIST,
+  IN_SUBSIDIARY,
+  USER_TYPE_ENTERPRISE,
 } from '../new-account-form-component.constants';
 
 export default class NewAccountFormFieldController {
   /* @ngInject */
-  constructor($filter, $locale, $scope, $timeout, $translate, atInternet) {
+  constructor(
+    $filter,
+    $locale,
+    $scope,
+    $timeout,
+    $translate,
+    atInternet,
+    coreConfig,
+  ) {
     this.$filter = $filter;
     this.$locale = $locale;
     this.$scope = $scope;
     this.$timeout = $timeout;
     this.$translate = $translate;
     this.atInternet = atInternet;
     this.FIELD_NAME_LIST = FIELD_NAME_LIST;
+    this.coreConfig = coreConfig;
   }
 
   $onInit() {
+    // Indian subsidiary changes
+    this.isIndianSubsidiary =
+      this.coreConfig.getUser().ovhSubsidiary === IN_SUBSIDIARY;
```

Comment:
**nitpick:** Instead of checking the sub both in `new-account-form-controller` and `new-account-form-field-component.controller`. The info can be passed to `new-account-form-field-component.controller` as an input.

Reply:
Done Anoop Thanks !!